core/store/tx_manager.go

```diff
@@ -230,7 +236,7 @@ func (txm *EthTxManager) createTx(
 	gasLimit uint64,
 	value *assets.Eth) (*models.Tx, error) {
 
-	for nrc := 0; nrc <= nonceReloadLimit; nrc++ {
+	for nrc := 0; nrc < nonceReloadLimit+1; nrc++ {
```

Comment:
why + 1?

Reply:
We want the loop to run at least once, and then `nonceReloadLimit` many times. So even if nonceReloadLimit is zero , it still runs at least once.

Put differently, it's nonce**Re**loadLimit, not nonceLoadLimit.

Reply:
`+1` rather than `<=` (they are equivalent) is for greater clarity